recipes/gudhi/meta.yaml

```diff
@@ -0,0 +1,59 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "gudhi" %}
+{% set version = "2.1.0" %}
+{% set url = "https://gforge.inria.fr/frs/download.php/file/37362/2018-01-31-09-25-53_GUDHI_2.1.0.tar.gz" %}
+{% set sha256 = "820f4fccbe3ff5a313de947b653667d5b01cea0f38fc6ba6ab09d633e529e03d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  url: {{ url }}
+build:
+  number: 0
+  features:
+    - vc14  # [win and py36]
+  # C++11 is supported from vc14 and py36
+  skip: true  # [py<36]
+requirements:
+  build:
+    - boost-cpp 1.66.0
+    - cgal 4.11
+    - cmake
+    - cython
+    - eigen
+    - mpfr 3.1.*
+    - mpir 3.0.0
+    - python
+    - matplotlib
+    - numpy 1.13.*
+    - pytest
+    - sphinx
+    - sphinxcontrib-bibtex 
+    - toolchain
+  run:
+    - boost-cpp 1.66.0
+    - cgal 4.11
+    - mpfr >=3.1
+    - python
+    - numpy >=1.13
+    - matplotlib
+test:
+  imports:
+    - gudhi
+about:
+  home: http://gudhi.gforge.inria.fr/
+  license: GPL3
```

Comment:
This needs a license file and a license .

Reply:
Ok. I changed it on commit b78bb80